ibis/expr/format.py

```diff
@@ -16,6 +18,7 @@ def __init__(self):
         self._repr_memo = {}
         self.subexprs = {}
         self.visit_memo = set()
+        self.get_text_repr = False
```

Comment:
Forgot in the previous review, but you'll have to get this as a parameter of the constructor. Also, if you don't mind adding a proper docstring to this class, also including the new parameter. This class is quite tricky, and since you're familiar with it now, it'd be great if you could help future readers. Thanks!